.github/workflows/cache.yml

```diff
@@ -209,3 +222,47 @@ jobs:
           path: |
             /home/runner/.local/share/bi/dev/
           key: ${{ needs.check-caches.outputs.bi-hash-key }}
+
+  cache-images:
+    if: ${{ ! (needs.check-caches.outputs.image-cache-hit) }}
+    needs: check-caches
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    # we'll want to pull existing base images if possible
+    permissions:
+      id-token: write
+      contents: read
+      packages: read
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          persist-credentials: false
+
+      - name: Setup ASDF
+        uses: ./.github/actions/setup-asdf
+
+      - name: Login to GHCR
+        uses: docker/login-action@74a5d142397b4f367a81961eba4e8cd7edddf772 # v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build current images
+        run: |
+          bin/bix docker ensure-base-images
+          bin/bix docker build-image control-server
+          bin/bix docker build-image kube-bootstrap
+          bin/bix docker retag-images-for-ghcr
+
+      - name: Create image archive
+        run:
+          bin/bix registry save-int-images
+          /home/runner/work/_temp/base_images.tar
```

Comment:
This includes the control-server and kube-bootstrap images, which means this shared cache will have a lot of overwrites. Should we add a `bin/bix registry save-base-int-images` command that includes the base image and the registry images?

Bascially I am worried this cache is going to leak data and become a write bottleneck.

Reply:
Well, what I've currently got will only update if the registry changes or the dockerfiles change (basically we've updated the base images). It'd help to already have those layers present as it's less to pull and extract from the registry.

I'm not opposed to having a separate command for just the base images though. I had definitely thought about it.